packages/utils/src/validation/validatorFunctions.js

```diff
@@ -87,6 +87,14 @@ export const isPlainObject = value => {
   }
 };
 
+export const assertAllValuesAreNumbers = object => {
+  Object.entries(object).forEach(([key, value]) => {
+    if (typeof value !== 'number') {
+      throw new Error(`Value '${key}' is not a number: '${value}'`);
```

Comment:
For consistency with the rest of the validator functions, although we will change this at some point according to https://github.com/beyondessential/tupaia-backlog/issues/1201:

```suggestion
      throw new ValidationError(`Value '${key}' is not a number: '${value}'`);
```